sources/packages/backend/workflow/test/test-utils/execute-workflow-instance-utils.ts

```diff
@@ -59,3 +59,17 @@ export async function executePartTimeAssessmentForProgramYear(
     assessmentConsolidatedData,
   );
 }
+
+/**
+ * Executes the parttime-configure-disbursement BPMN workflow.
+ * @param assessmentConsolidatedData assessment data.
+ * @returns result of the workflow execution.
+ */
+export async function executePartTimeConfigureDisbursement(
```

Comment:
I believe the `parttime-configure-disbursement` uses a limited number of variables. Can we please reduce the amount of variables to what is used for the workflow?
I would recommend creating a separated interface or a partial interface from`AssessmentConsolidatedData` adding any extra property needed.